Npm install fails

see https://github.com/iurimatias/embark-framework/issues/99

Here is the output of `npm install secp256k1@1.1.4 -g`:

```
> secp256k1@1.1.4 install /usr/local/lib/node_modules/secp256k1
> node-gyp rebuild

  CXX(target) Release/obj.target/secp256k1/functions.o
../functions.cc:224:57: warning: variable 'pub_key' is uninitialized when used here [-Wuninitialized]
  int results = secp256k1_ec_pubkey_parse(secp256k1ctx, pub_key, pk_data, pub_len);
                                                        ^~~~~~~
../functions.cc:223:30: note: initialize the variable 'pub_key' to silence this warning
  secp256k1_pubkey_t *pub_key;
                             ^
                              = NULL
../functions.cc:248:56: warning: variable 'pub_key' is uninitialized when used here [-Wuninitialized]
  int results= secp256k1_ec_pubkey_parse(secp256k1ctx, pub_key, pk_data, pub_len);
                                                       ^~~~~~~
../functions.cc:247:30: note: initialize the variable 'pub_key' to silence this warning
  secp256k1_pubkey_t *pub_key;
                             ^
                              = NULL
2 warnings generated.
  CXX(target) Release/obj.target/secp256k1/util.o
  CXX(target) Release/obj.target/secp256k1/secp256k1.o
  CC(target) Release/obj.target/secp256k1/secp256k1-src/src/secp256k1.o
In file included from ../secp256k1-src/src/secp256k1.c:12:
In file included from ../secp256k1-src/src/num_impl.h:17:
../secp256k1-src/src/num_gmp_impl.h:25:13: warning: unused function 'secp256k1_num_copy'
      [-Wunused-function]
static void secp256k1_num_copy(secp256k1_num_t *r, const secp256k1_num_t *a) {
            ^
../secp256k1-src/src/num_gmp_impl.h:163:12: warning: unused function 'secp256k1_num_eq'
      [-Wunused-function]
static int secp256k1_num_eq(const secp256k1_num_t *a, const secp256k1_num_t *b) {
           ^
../secp256k1-src/src/num_gmp_impl.h:195:13: warning: unused function 'secp256k1_num_add'
      [-Wunused-function]
static void secp256k1_num_add(secp256k1_num_t *r, const secp256k1_num_t *a, const secp256k1_n...
            ^
../secp256k1-src/src/num_gmp_impl.h:201:13: warning: unused function 'secp256k1_num_sub'
      [-Wunused-function]
static void secp256k1_num_sub(secp256k1_num_t *r, const secp256k1_num_t *a, const secp256k1_n...
            ^
../secp256k1-src/src/num_gmp_impl.h:207:13: warning: unused function 'secp256k1_num_mul'
      [-Wunused-function]
static void secp256k1_num_mul(secp256k1_num_t *r, const secp256k1_num_t *a, const secp256k1_n...
            ^
../secp256k1-src/src/num_gmp_impl.h:80:13: warning: unused function 'secp256k1_num_mod'
      [-Wunused-function]
static void secp256k1_num_mod(secp256k1_num_t *r, const secp256k1_num_t *m) {
            ^
../secp256k1-src/src/num_gmp_impl.h:234:13: warning: unused function 'secp256k1_num_shift'
      [-Wunused-function]
static void secp256k1_num_shift(secp256k1_num_t *r, int bits) {
            ^
../secp256k1-src/src/num_gmp_impl.h:149:12: warning: unused function 'secp256k1_num_is_neg'
      [-Wunused-function]
static int secp256k1_num_is_neg(const secp256k1_num_t *a) {
           ^
../secp256k1-src/src/num_gmp_impl.h:256:13: warning: unused function 'secp256k1_num_negate'
      [-Wunused-function]
static void secp256k1_num_negate(secp256k1_num_t *r) {
            ^
In file included from ../secp256k1-src/src/secp256k1.c:13:
In file included from ../secp256k1-src/src/field_impl.h:19:
../secp256k1-src/src/field_5x52_impl.h:54:13: warning: unused function 'secp256k1_fe_verify'
      [-Wunused-function]
static void secp256k1_fe_verify(const secp256k1_fe_t *a) {
            ^
In file included from ../secp256k1-src/src/secp256k1.c:15:
../secp256k1-src/src/group_impl.h:36:13: warning: unused function 'secp256k1_ge_set_infinity'
      [-Wunused-function]
static void secp256k1_ge_set_infinity(secp256k1_ge_t *r) {
            ^
../secp256k1-src/src/group_impl.h:161:13: warning: unused function 'secp256k1_gej_set_xy'
      [-Wunused-function]
static void secp256k1_gej_set_xy(secp256k1_gej_t *r, const secp256k1_fe_t *x, const secp256k1...
            ^
In file included from ../secp256k1-src/src/secp256k1.c:14:
../secp256k1-src/src/scalar_impl.h:28:13: warning: unused function 'secp256k1_scalar_get_num'
      [-Wunused-function]
static void secp256k1_scalar_get_num(secp256k1_num_t *r, const secp256k1_scalar_t *a) {
            ^
In file included from ../secp256k1-src/src/secp256k1.c:14:
In file included from ../secp256k1-src/src/scalar_impl.h:20:
../secp256k1-src/src/scalar_4x64_impl.h:886:13: warning: unused function 'secp256k1_scalar_split_128'
      [-Wunused-function]
static void secp256k1_scalar_split_128(secp256k1_scalar_t *r1, secp256k1_scalar_t *r2, const ...
            ^
In file included from ../secp256k1-src/src/secp256k1.c:15:
../secp256k1-src/src/group_impl.h:227:12: warning: unused function 'secp256k1_gej_is_valid_var'
      [-Wunused-function]
static int secp256k1_gej_is_valid_var(const secp256k1_gej_t *a) {
           ^
15 warnings generated.
  SOLINK_MODULE(target) Release/secp256k1.node
  SOLINK_MODULE(target) Release/secp256k1.node: Finished
secp256k1@1.1.4 /usr/local/lib/node_modules/secp256k1
├── bindings@1.2.1
└── nan@2.1.0
```

--- a/binding.gyp
+++ b/binding.gyp
@@ -47,6 +47,7 @@
       "-Wextra"
     ],
     "include_dirs": [
+      "/usr/local/include",
       "./secp256k1-src",
       "<!(node -e \"require('nan')\")"
     ],
@@ -92,11 +93,8 @@
       ],
       [
        'OS=="mac"', {
-          "defines": [
-            "HAVE_LIBGMP=0",
-            "USE_NUM_GMP=0",
-            "USE_FIELD_INV_NUM=0",
-            "USE_SCALAR_INV_NUM=0"
+          "libraries": [
+            "-L/usr/local/lib"
           ]
       }],
       [

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "secp256k1",
-  "version": "1.1.4",
+  "version": "1.1.5",
   "description": "This module provides native bindings to ecdsa secp256k1 functions",
   "main": "index.js",
   "scripts": {

